RetsTalk/RetsTalk/Retrospect/Controller/RetrospectListViewController.swift

```diff
@@ -163,9 +163,12 @@ final class RetrospectListViewController: BaseViewController {
     
     private func updateTotalRetrospectCount() {
         Task {
-            guard let count = await retrospectManager.fetchRetrospectsCount() else { return }
+            guard let fetchedCount = await retrospectManager.fetchRetrospectsCount() else { return }
```

Comment:
지금것도 좋지만 어떤걸 패치해온 개수인지 알 수 있으면 더 좋을 것 같아요!
```suggestion
            guard let fetchedRetrospectsCount = await retrospectManager.fetchRetrospectsCount() else { return }
```